CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Enhance CORS config options
```

Comment:
Do we want v2 changes to be included in the changelog?

Reply:
@inlined Suggested adding a changelog to this, but it makes sense not to have v2 changes in there